spec/models/manageiq/providers/kubernetes/container_manager/metrics_capture/capture_context_spec.rb

```diff
@@ -0,0 +1,90 @@
+describe ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture::CaptureContext do
+  before(:each) do
+    allow(MiqServer).to receive(:my_zone).and_return("default")
+    hostname = 'ose-master.qa.lab.com'
+    hostname_hawkular = 'ose-node1.qa.lab.com'
+    token = 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ'\
+      'pc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW5'\
+      '0Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3V'\
+      'udC9uYW1lc3BhY2UiOiJtYW5hZ2VtZW50LWluZnJ'\
+      'hIiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3V'\
+      'udC9zZWNyZXQubmFtZSI6Im1hbmFnZW1lbnQtYWR'\
+      'taW4tdG9rZW4tdXI2YW0iLCJrdWJlcm5ldGVzLml'\
+      'vL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3V'\
+      'udC5uYW1lIjoibWFuYWdlbWVudC1hZG1pbiIsImt'\
+      '1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2V'\
+      'ydmljZS1hY2NvdW50LnVpZCI6IjYxNWU2MWRkLTM'\
+      'yMTMtMTFlNi1iMzg3LTAwMWE0YTE2OTc5OSIsInN'\
+      '1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDptYW5'\
+      'hZ2VtZW50LWluZnJhOm1hbmFnZW1lbnQtYWRtaW4'\
+      'ifQ.lnv42HrZ3hujxG8kIovb89XiOVdJEY83FU6h'\
+      'sfA5RFK05gMI8vqBStJmFYpCTRScrBXkljajDKX7'\
+      'OT83mhZ6JoWkfJtDAul4NE7tV1NL_yr1IT-YwNY0'\
+      'eCD_BjvJw-H_JdEDo0ivyByAHnfJOB7GK27a6DtO'\
+      'Yu4aR5eqlV7kQb3zusgNcAScEeYJcVDHWbGgr0hB'\
+      'c1E7Zc4EvvjZWOmv_S7H5RxpZfXkPvOr20-rEpaj'\
+      '0RJ2hs7nxZv4ahuSPUlwzSQdRj_BQss2iGy7mbSp'\
+      'gBbHTTagOHwhrZmMSgwfwpzPAiFij42xF61HCK_Z'\
+      'LrDEZ7_HDfHAXRa0pHEbh4D2IQ'
+
+    @ems = FactoryGirl.create(
+      :ems_openshift,
+      :name                      => 'QaLab',
+      :connection_configurations => [{:endpoint       => {:role     => :default,
+                                                          :hostname => hostname,
+                                                          :port     => "8443"},
+                                      :authentication => {:role     => :bearer,
+                                                          :auth_key => token,
+                                                          :userid   => "_"}},
+                                     {:endpoint       => {:role     => :hawkular,
+                                                          :hostname => hostname_hawkular,
+                                                          :port     => "443"},
+                                      :authentication => {:role     => :hawkular,
+                                                          :auth_key => token,
+                                                          :userid   => "_"}}])
+  end
+
+  it "will read hawkular status (does not require authentication)" do
+    VCR.use_cassette("#{described_class.name.underscore}_status") do # , :record => :new_episodes) do
+      EmsRefresh.refresh(@ems)
+      context = ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture::CaptureContext.new(
+        @ems.container_nodes.last, 1_466_505_130, 1_466_508_726, 20)
```

Comment:
Consider for clarity:
``
require 'date'
#...
timestamp_fmt = "%s"
start = DateTime.parse("2016-06-21T10:32:10").strftime(timestamp_fmt).to_i
end = DateTime.parse("2016-06-21T11:32:06").strftime(timestamp_fmt).to_i
duration = 20
```
(these are the actual timestamps you used.)

When re-rerecording, I suppose one would need to update to close-to-now times.